pkg/cmd/server/admin/create_bootstrap_project_template.go

```diff
@@ -30,7 +34,11 @@ func NewCommandCreateBootstrapProjectTemplate(f *clientcmd.Factory, commandName
 				cmdutil.CheckErr(cmdutil.UsageError(cmd, err.Error()))
 			}
 
-			template, err := options.CreateBootstrapProjectTemplate()
+			// We can't depend on len(options.NodeSelector) > 0 as node-selector="" is valid
+			// and we want to populate node selector as project annotation only if explicitly set by user
+			checkNodeSelector := cmd.Flag("node-selector").Changed
```

Comment:
I'm against options.  I'd say that if the user wants something other than our vanilla template, customizing it is up to him.  @liggitt ?